specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/CommonDefinitions.json

```diff
@@ -1176,6 +1180,20 @@
         }
       }
     },
+    "KeyVaultProperties": {
+      "description": "Key vault reference using managed identity.",
```

Comment:
>Key vault reference using managed identity. [](http://example.com/codeflow?start=22&length=43)

Properties of Certificate stored in a Key Vault?

Reply:
The design is fine, but my suggestion is primarily on naming of these definitions. As I understood, your "keyVaultUrl" should be named "keyVaultSecretUrl" (or something similar), as this URL represents URL for a secret, not for Key Vault itself. I hope that makes sense.

Reply:
Also, description "Key vault reference using managed identity." for a definition named "KeyVaultProperties" doesn't seem appropriate.